packages/next/src/generators/application/files/app/page.tsx__tmpl__

```diff
@@ -14,7 +14,7 @@
 const StyledPage = styled.div`<%- pageStyleContent %>`;
 <% }%>
 
-export default async function Index() {
+export default function Index() {
```

Comment:
Let's add back the `async` since it is a server component. 
Users would be already setup to fetch data.